src/browser.ts

```diff
@@ -33,7 +33,12 @@ export class Browser {
       // set env timezone
       env.TZ = options.timezone || process.env.TZ;
 
-      browser = await puppeteer.launch({env: env, args: ['--no-sandbox']});
+      const executablePath = puppeteer.executablePath().replace(/^.*?\/node_modules/, path.dirname(process.execPath));
```

Comment:
As puppeteer isn't baked in the path needs to be changed here. There is `process.pkg` property that can be checked so this still works outside of the binary.